plugins/modules/nmcli.py

```diff
@@ -1460,14 +1477,18 @@
     autoconnect: true
     state: present
 
-- name: Create a wireguard connection
+- name: Create a wireguard connection with peer configuration
   community.general.nmcli:
     type: wireguard
     conn_name: my-wg-provider
     ifname: mywg0
     wireguard:
         listen-port: 51820
         private-key: my-private-key
+    wgpeer_public_key: "your_peer_public_key"
```

Comment:
This looks wrongly indendet. According to the documentation, this belongs in the `wireguard` dict, and not to the top-level.
```suggestion
        wgpeer_public_key: "your_peer_public_key"
```
Same below.

Reply:
ok let me fix this

Reply:
fixed the identation